online_test/.env.sample

```diff
@@ -0,0 +1,16 @@
+# Django settings
+
+SECRET_KEY=dUmMy_s3cR3t_k3y
+#DB_ENGINE=mysql
+#DB_NAME=name_of_db
+#DB_USER=name_of_db_user
+#DB_PASSWORD=db_password
+#DB_HOST=db_hostname
+#DB_PORT=8180
+
+# Yaksh settings
+
+#N_CODE_SERVERS=5
+#SERVER_POOL_PORT=55555
+#SERVER_HOST_NAME=http://localhost
+#SERVER_TIMEOUT=4
```

Comment:
Is this necessary? Since you have also added `.sampleenv`.

Reply:
Fixed